corehq/apps/hqadmin/forms.py

```diff
@@ -114,6 +96,81 @@ def __init__(self, can_toggle_is_staff, *args, **kwargs):
         )
 
 
+class OffboardingUserListForm(forms.Form):
+    csv_email_list = forms.CharField(
+        label="Comma seperated email addresses",
+        widget=forms.Textarea()
+    )
+    invert_apply = forms.BooleanField(
+        label="Invert",
+        initial=False,
+        required=False,
+        help_text=(
+            "Get dimagi.com accounts not included in the above, comma-separated list."
+        )
+    )
```

Comment:
I think for the new offboarding page, "Invert" should just be the default behavior and not a separate option. I don't think we'd ever use that page to look up a list of specific people. That's what "Grant superuser privileges" and "Lookup user by email" are for.